types/feathersjs__express/index.d.ts

```diff
@@ -10,12 +11,18 @@ import * as express from 'express';
 export default function feathersExpress<T>(app: FeathersApplication<T>): Application<T>;
 export type Application<T> = express.Application & FeathersApplication<T>;
 
-export function errorHandler(options?: any): express.ErrorRequestHandler;
+export function errorHandler(options?: {
+    public?: string,
+    logger?: { error?: (msg: string) => void },
+    html?: any,
+    json?: any,
+}): express.ErrorRequestHandler;
 export function notFound(): express.RequestHandler;
-export const rest: {
-    (): () => void;
-    formatter: express.RequestHandler;
-};
+
+export function rest(): (handler?: express.RequestHandler) => void;
```

Comment:
This is not the actual behavior. `rest` takes the handler as an optional argument and returns a `configure`able function, which is `() => void`